Contributing.md

```diff
@@ -29,13 +29,16 @@ If you're reading this, you're probably creating a Pull Request or planning to d
    [Install](https://docs.docker.com/engine/install/)
 
```

Comment:
Write here:
After installing docker, run the following command to start database containers.